airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/engagements_emails.json

```diff
@@ -5,255 +5,250 @@
-        },
-        "hs_email_status": {
-          "type": ["null", "string"]
-        },
-        "hs_email_subject": {
-          "type": ["null", "string"]
-        },
-        "hs_email_text": {
-          "type": ["null", "string"]
-        },
-        "hs_email_thread_id": {
-          "type": ["null", "string"]
-        },
-        "hs_email_to_email": {
-          "type": ["null", "string"]
-        },
-        "hs_email_to_firstname": {
-          "type": ["null", "string"]
-        },
-        "hs_email_to_lastname": {
-          "type": ["null", "string"]
-        },
-        "hs_email_to_raw": {
-          "type": ["null", "string"]
-        },
-        "hs_email_tracker_key": {
-          "type": ["null", "string"]
-        },
-        "hs_email_validation_skipped": {
-          "type": ["null", "string"]
-        },
-        "hs_engagement_source": {
-          "type": ["null", "string"]
-        },
-        "hs_engagement_source_id": {
-          "type": ["null", "string"]
-        },
-        "hs_follow_up_action": {
-          "type": ["null", "string"]
-        },
-        "hs_gdpr_deleted": {
-          "type": ["null", "boolean"]
-        },
-        "hs_lastmodifieddate": {
-          "type": ["null", "string"],
-          "format": "date-time"
-        },
-        "hs_merged_object_ids": {
-          "type": ["null", "string"]
-        },
-        "hs_modified_by": {
-          "type": ["null", "string"]
-        },
-        "hs_object_id": {
-          "type": ["null", "number"]
-        },
-        "hs_product_name": {
-          "type": ["null", "string"]
-        },
-        "hs_queue_membership_ids": {
-          "type": ["null", "string"]
-        },
-        "hs_timestamp": {
-          "type": ["null", "string"],
-          "format": "date-time"
-        },
-        "hs_unique_creation_key": {
-          "type": ["null", "string"]
-        },
-        "hs_unique_id": {
-          "type": ["null", "string"]
-        },
-        "hs_updated_by_user_id": {
-          "type": ["null", "number"]
-        },
-        "hs_user_ids_of_all_notification_followers": {
-          "type": ["null", "string"]
-        },
-        "hs_user_ids_of_all_notification_unfollowers": {
-          "type": ["null", "string"]
-        },
-        "hs_user_ids_of_all_owners": {
-          "type": ["null", "string"]
-        },
-        "hubspot_owner_assigneddate": {
-          "type": ["null", "string"],
-          "format": "date-time"
-        },
-        "hubspot_owner_id": {
-          "type": ["null", "string"]
-        },
-        "hubspot_team_id": {
-          "type": ["null", "string"]
-        },
-        "hs_all_owner_ids": {
-          "type": ["null", "string"]
-        },
-        "hs_all_team_ids": {
-          "type": ["null", "string"]
-        },
-        "hs_all_accessible_team_ids": {
-          "type": ["null", "string"]
-        }
-      }
+    "properties_hs_all_assigned_business_unit_ids": {
```

Comment:
not sure if natalie corrected this, but i did noticed you asked. I assume it was a typo and she meant properties as you have it

Reply:
if I understand this right, this column is named correctly. Please check out [this](https://github.com/airbytehq/airbyte/issues/29345#issuecomment-1718292126) answer and see if it answers your question